app/mailers/oral_history_delivery_mailer.rb

```diff
@@ -0,0 +1,70 @@
+class OralHistoryDeliveryMailer < ApplicationMailer
+
+  def oral_history_delivery_email
+    raise ArgumentError.new("Required params[:request] missing") unless request.present?
+    raise ArgumentError.new("Required patron email missing") unless request.patron_email.present?
+    mail(to: to_address, subject: subject, content_type: "text/html")
+  end
+
+  def request
+    @request ||= params[:request]
+  end
+
+  def to_address
+    request.patron_email
+  end
+
+  # Note: any value greater than 604800 will raise an exception.
+  # See https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/S3/Presigner.html
+  def how_long_urls_will_be_valid
+    1.week.to_i - 10
+  end
```

Comment:
How about calling this method `asset_expiration_time` instead? `how_long_` rubs me funny as a name. 

And I'm suspicious of that `-10`?  If it raises an error without the `- 10`, that might be a bug in my FasterS3Urls gem, or something else, that I want to get to the bottom of.

Reply:
Fixed method name.

 I do *not* believe your `FasterS3Urls` has a bug.
I did some basic testing in the staging console that indicated this should work.
https://github.com/sciencehistory/scihist_digicoll/pull/901#issuecomment-714528948

Reply:
Got rid of the - 10. Works fine.